packages/lesswrong/components/review/ReviewVotingPage.tsx

```diff
@@ -286,12 +299,30 @@ const ReviewVotingPage = ({classes}: {
 
   const { LWTooltip, Loading, ReviewVotingExpandedPost, ReviewVoteTableRow, SectionTitle, RecentComments, FrontpageReviewWidget } = Components
 
-  const reSortPosts = useCallback(() => {
+  const reSortPosts = useCallback((sortPostsMethod, sortReversed) => {
     if (!postsResults) return
+
     const randomPermutation = generatePermutation(postsResults.length, currentUser)
     const newlySortedPosts = postsResults
       .map((post, i) => ([post, randomPermutation[i]] as const))
-      .sort(([post1, permuted1], [post2, permuted2]) => {
+      .sort(([inputPost1, permuted1], [inputPost2, permuted2]) => {
+        const post1 = sortReversed ? inputPost2 : inputPost1
+        const post2 = sortReversed ? inputPost1 : inputPost2
+
+        if (sortPostsMethod === "needsReview") {
+          // This prioritizes posts with no reviews, which you highly upvoted
+          const post1NeedsReview = post1.reviewCount === 0 && post1.reviewVoteScoreHighKarma > 4
+          const post2NeedsReview = post2.reviewCount === 0 && post2.reviewVoteScoreHighKarma > 4
```

Comment:
Are these supposed to check if the **current user** has voted them highly? Isn't `reviewVoteScoreHighKarma` something like, the sum of votes from high-karma users?

Reply:
The idea here is

a) filter for posts that have a shot in hell at winning, which have not yet been reviewed
b) sort those by the currentUser's vote.

The goal is to give the user a (somewhat) manageable list of things to consider reviewing.